osgeoproj_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export URL='https://services1.arcgis.com/dePSdaG71BplHMCO/arcgis/rest/services/TaPS_Prkg_Zones/FeatureServer/0/query/?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=*&returnGeometry=true&returnCentroid=true&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&returnIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnDistinctValues=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token='
```

Comment:
Not that this API is going anywhere, but would it make more sense to store this data locally? Just so this test does not have any dependency on the ArcGIS API.

Reply:
Hmm thats a good idea but considering we only use this with ArcGIS for now we can leave it as is.

I'm going to be refactoring this script into the Jenkins/Ansible role as well.